lib/services/test-execution-service.ts

```diff
@@ -128,22 +130,33 @@ class TestExecutionService implements ITestExecutionService {
 					this.$errors.failWithoutHelp("Verify that listed files are well-formed and try again the operation.");
 				}
 
-				const deployOptions: IDeployPlatformOptions = {
-					clean: this.$options.clean,
-					device: this.$options.device,
-					emulator: this.$options.emulator,
-					projectDir: this.$options.path,
-					platformTemplate: this.$options.platformTemplate,
-					release: this.$options.release,
-					provision: this.$options.provision,
-					teamId: this.$options.teamId
-				};
-
-				const buildConfig: IBuildConfig = _.merge({ buildForDevice: this.$options.forDevice }, deployOptions);
-
 				if (this.$options.debugBrk) {
-					await this.getDebugService(platform).debug(projectData, buildConfig);
+					const debugService = this.getDebugService(platform); const deployOptions: IDeployPlatformOptions = {
```

Comment:
`deployOptions` can be declared above the `if` statement in order to avoid repetition